libgnucash/engine/gnc-date.cpp

```diff
@@ -201,6 +201,12 @@ gnc_gmtime (const time64 *secs)
 
 }
 
+gint
+start_of_week ()
+{
+    return *nl_langinfo(_NL_TIME_FIRST_WEEKDAY);
```

Comment:
Will this be portable to Macos and Windows ?

Reply:
No idea! From stack overflow it'd seem there's published API for ICU only. I don't have Mac, and haven't attempted windows build yet. So perhaps we try ICU instead? (Which did work according to previous excitement)